src/iuse.cpp

```diff
@@ -1841,6 +1841,31 @@ int iuse::purify_iv(player *p, item *it, bool, const tripoint& )
     return it->type->charges_to_use();
 }
 
+void spawn_spores( player *p ) {
+    int spore_spawned = 0;
+    for ( int x = p->posx() - 4; x <= p->posx() + 4; x++ ) {
+        for ( int y = p->posy() - 4; y <= p->posy() + 4; y++ ) {
```

Comment:
Can (should) be changed to `for( const tripoint &dest : g->m.points_in_radius( p.pos(), 4 ) )`